packages/squiggle-lang/src/rescript/Reducer/Reducer.res

```diff
@@ -1,27 +1,35 @@
 module ErrorValue = Reducer_ErrorValue
 module Expression = Reducer_Expression
-module ExpressionValue = ReducerInterface_ExpressionValue
+module ExternalExpressionValue = ReducerInterface_ExpressionValue
```

Comment:
Can we rename ``ReducerInterface_ExpressionValue`` to ``ReducerInterface_ExternalExpressionValue`` as well, to be more clear?